core/GlobalState.cc

```diff
@@ -104,6 +104,13 @@ struct MethodBuilder {
         return *this;
     }
 
+    MethodBuilder &repeatedTopArg(NameRef name) {
+        auto &arg = gs.enterMethodArgumentSymbol(Loc::none(), method, name);
+        arg.flags.isRepeated = true;
+        arg.type = Types::top();
+        return *this;
+    }
```

Comment:
This is to make `args` into `*args` basically right?

Reply:
Yep.